content/docs/overview.md

```diff
@@ -160,7 +160,7 @@ For more information regarding value declarations in general, please see the [Od
 
 
 ## Packages
-Every Odin program is made up of packages. Programs begin running in the package `main`.
+Every Odin program is made up of packages. A package is a directory of Odin code files, with each file in that directory having the same package declaration at the top. Programs begin running in the package `main`.
```

Comment:
I'd condense this a bit:

Odin programs consist of packages. A package is a directory of Odin code files, all of which have the same package declaration at the top. Execution starts in the package's `main` procedure.

Reply:
Taken this wording as is. Thank you.